.github/pull_request_template.md

```diff
@@ -1,15 +1,52 @@
-Describe your changes here.
+_Authors: Please fill out this form carefully and completely._
 
-Closes # (type the issue number after # if applicable; otherwise remove this line)
+_Reviewers: By approving this Pull Request you are approving the code change, as well as its deployment and mitigation plans._
+_Please read this description carefully. If you feel there is anything unclear or missing, please ask for updates._
+
+#### List the issues that this change affects.
+<!--Every code change must address _at least 1_ issue. Fixes a bug, completes a task, every change
+      should have a corresponding issue listed here. If one does not already exist, create one. -->
+      
+Closes # (type the GitHub issue number after #)
+
+### What are you trying to accomplish?
+<!-- Provide a description of the changes. -->
 
 ### Screenshots
+<!-- Provide before/after screenshots, videos, or graphs for any visual changes; otherwise, remove this section -->
+
+#### Risk Assessment
+  <!-- Please select from one of the following and detail why this level was chosen -->
+
+- [ ] **Low risk** the change is small, highly observable, and easily rolled back.
+- [ ] **Medium risk** changes that are isolated, reduced in scope or could impact few users. The change will not impact library availability.
+- [ ] **High risk** changes are those that could impact customers and SLOs, low or no test coverage, low observability, or slow to rollback.
+
+### What approach did you choose and why?
+<!-- This section is a place for you to describe your thought process in making these changes.
+     List any tradeoffs you made to take on or pay down tech debt.
+     Identify any work you did to mitigate risk.
+     Describe any alternative approaches you considered and why you discarded them. -->
```

Comment:
This seems like something that can be rolled into the overview section above

```suggestion
```

Reply:
Removed this section in https://github.com/primer/react/pull/3344/commits/8fb13a10eb835d35835123fbcefa12a572c96e8c